modules/op-resource-quota-for-pipelines.adoc

```diff
@@ -0,0 +1,205 @@
+// This module is included in the following assembly:
+//
+// */cicd/pipelines/reducing-pipelines-resource-consumption.adoc
+
+[id="resource-quota-for-pipelines_{context}"]
+= Resource quota for pipelines
+
+A resource quota limits the total resource consumption per namespace. It can limit the quantity of objects created in a namespace, based on the type of the object. In addition, it can limit the total amount of compute resources consumed by resources in that namespace, using a compute resource quota.
```

Comment:
Let's mention that we are explicitly talking about the **`ResourceQuota` object and let's link to Kubernetes resource quota docs as well.